tests/trello/gui/resources/components/TopBar.resource

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Documentation    Variables and Keywords for Trello's Board Page Top menu bar
+...              (default page after logging in, the top bar, where the title of the board is located)
```

Comment:
If this is the menu at the top of a board page... then it's not in the default page after logging in...
We should maybe think of another name, since "Bar" isn't very descriptive... and also there's another menu on top of this one...